ProjectEureka/project-eureka-flutter
# PE-44 | Implement more details page

What i did:
Added new page .  When user clicks on more details under the question  from Home screen , it takes to new page with question details.

How to test:
- git pull
- switch to branch 44
- git pull
- git fetch
- flutter pub get

Run code, and after login, on Home page click on any  `More details` button. 
You should be taken to new page with question details.
Answer/Archive question button as of right now just takes user to Home page.


-------------------------------------------------------------------
Update
-------------------------------------------------------------------
To see different buttons, please edit line 209
![image](https://user-images.githubusercontent.com/64664704/112591076-09598b80-8dc1-11eb-9413-224a2a1ab35f.png)



change  !=  to  == 
if (user_id != owner_id) 
You should be able to see Archive button or Answer button
